pinc/showavailablebooks.inc

```diff
@@ -243,14 +243,14 @@ function show_projects_for_pool( $pool, $checkedout_or_available )
 
     $anchor = $checkedout_or_available;
 
-    $columns = array(
-        "nameofwork"          => array( 'Title',  _("Title") ),
-        "authorsname"         => array( 'Author', _("Author") ),
-        "language"            => array( 'Lang',   _("Language") ),
-        "genre"               => array( 'Genre',  _("Genre") ),
-        "n_pages"             => array( 'PgTot',  _("Pages") ),
-        $pool->foo_field_name => array( 'Person', $pool->foo_Header ),
-    );
+    $columns = [
+        "nameofwork" => ['Title', _("Title")],
+        "authorsname" => ['Author', _("Author")],
+        "language" => ['Lang', _("Language")],
+        "genre" => ['Genre', _("Genre")],
+        "n_pages" => ['PgTot', _("Total Pages")],
```

Comment:
another functional change, right? Just making sure they're intentional.

Reply:
This was a copy/paste mistake - thank you for catching it!